pkg/csi/service/wcp/controller.go

```diff
@@ -18,8 +18,13 @@ package wcp
 
 import (
 	"fmt"
+	"net/http"
 	"path/filepath"
 	"strings"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
```

Comment:
Shouldn't this be at Line 29 or higher. Is grouping of imports not mandatory in PR's linter checks.

Reply:
Not sure. These are formatted before pushed to GitHub.